src/Aspire.Hosting.Redis/Aspire.Hosting.Redis.csproj

```diff
@@ -22,6 +22,7 @@
 
   <ItemGroup>
     <InternalsVisibleTo Include="Aspire.Hosting.Tests" />
+    <InternalsVisibleTo Include="Aspire.Hosting.Redis.Tests" />
```

Comment:
@sebastienros - can we remove this? What is it needed for?

Reply:
RedisContainerImageTags
RedisCommanderConfigWriterHook

Are use in these tests and internal.
This is why `Aspire.Hosting.Tests` was already listed because these tests are coming from (moved to this new `Aspire.Hosting.Redis.Tests`)